Keeping session alive at Tue Apr  1 12:37:35 PM UTC 2025
Traceback (most recent call last):
  File "/home/user/test/main.py", line 1, in <module>
    import telebot
ModuleNotFoundError: No module named 'telebot'
Traceback (most recent call last):
  File "/home/user/test/main.py", line 1, in <module>
    import telebot
ModuleNotFoundError: No module named 'telebot'
Keeping session alive at Tue Apr  1 12:42:35 PM UTC 2025
Keeping session alive at Tue Apr  1 12:47:35 PM UTC 2025
Keeping session alive at Tue Apr  1 12:52:35 PM UTC 2025
Keeping session alive at Tue Apr  1 12:57:35 PM UTC 2025
Keeping session alive at Tue Apr  1 01:02:35 PM UTC 2025
Keeping session alive at Tue Apr  1 01:07:35 PM UTC 2025
Keeping session alive at Tue Apr  1 01:12:35 PM UTC 2025
Keeping session alive at Tue Apr  1 01:17:35 PM UTC 2025
Keeping session alive at Tue Apr  1 01:22:35 PM UTC 2025
Keeping session alive at Tue Apr  1 01:27:35 PM UTC 2025
Keeping session alive at Tue Apr  1 01:32:35 PM UTC 2025
Keeping session alive at Tue Apr  1 01:37:35 PM UTC 2025
Keeping session alive at Tue Apr  1 01:42:35 PM UTC 2025
Keeping session alive at Tue Apr  1 01:47:35 PM UTC 2025
